x-pack/plugins/index_lifecycle_management/public/services/sort_table.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { sortBy } from 'lodash';
+
+const stringSort = (fieldName) => (item) => item[fieldName];
+const arraySort = (fieldName) => (item) => (item[fieldName] || []).length;
+
+const sorters = {
+  name: stringSort('name'),
+  coveredIndices: arraySort('coveredIndices'),
+};
+export const sortTable = (array = [], sortField, isSortAscending) => {
```

Comment:
Do we know how many policies will likely have? What's the most they're likely to have? If in the hundreds, then it may be worth using `InMemoryTable` instead of managing table state. This will eliminate a lot of code. Perhaps something to explore in the future.

Reply:
I would expect the list to remain extremely small.  I doubt most people would have more than 10.